packages/flutter_test/test/matchers_test.dart

```diff
@@ -1324,6 +1324,72 @@ void main() {
       expect(find.byType(Text), isNot(findsAtLeastNWidgets(3)));
     });
   });
+
+  group('findsOneWidget', () {
```

Comment:
To review the changes to these tests, take a look at [this commit](https://github.com/flutter/flutter/pull/127137/commits/b41b78bdb5637229d3d6f90155051d410fe826fa) which has just the changes and not the move to the `matchers_test.dart` file.